apps/studio/src/pages/dashboard/edit/index.tsx

```diff
@@ -69,18 +70,35 @@ const placeholder = {
 
 const EditPage: NextPageWithLayout = () => {
   const [isEditorOpen, setIsEditorOpen] = useState(true)
-  const [editorValue, setEditorValue] = useState(
-    JSON.stringify(placeholder, null, 2),
-  )
-  const [newEditorValue, setNewEditorValue] = useState({})
   const [editedSchema, setEditedSchema] = useState<any>(placeholder)
   const [isJSONValid, setIsJSONValid] = useState(true)
   const [isNewEditor, setIsNewEditor] = useState(false)
-  const [isCopied, setIsCopied] = useState(false)
 
+  const [isCopied, setIsCopied] = useState(false)
```

Comment:
nit: we can also do away with this copying functionality, it's playground-specific code.

Reply:
agreed